snykTask/src/__tests__/test-severity-threshold.ts

```diff
@@ -0,0 +1,23 @@
+import { isNotValidThreshold, Severity } from "../index";
+
+describe("isNotValidThreshold", () => {
+  const validSeverityThresholds = [
+    Severity.CRITICAL,
+    Severity.HIGH,
+    Severity.MEDIUM,
+    Severity.LOW
+  ];
+  test("returns true if invalid severity threshold", () => {
+    const isInvalid = isNotValidThreshold("hey");
+    expect(isInvalid).toBe(true);
+  });
+
+  describe.each(validSeverityThresholds)(
+    "isNotValidThreshold(%s)",
+    (level, expected) => {
```

Comment:
Something isn't quite right about this
![image](https://user-images.githubusercontent.com/44034094/118702405-dfca4880-b7e2-11eb-9746-276c297efded.png)

Try this:
```
  describe.each(validSeverityThresholds)(
    "isNotValidThreshold(%s)",
    (level) => {
      test(`returns false for '${level}'`, () => {
        expect(isNotValidThreshold(level)).toBe(false);
      });
    }
  );
```

Reply:
Oh right good spot, I've incorporated your suggestion